Source/APIClient/Request.swift

```diff
@@ -18,32 +18,18 @@ public enum HTTPMethod: String {
 public protocol Request {
     static var baseURL: String { get }
     static var acceptHeader: String? { get }
-    static var authToken: String? { get set }
+    static var authToken: String? { get }
```

Comment:
hahaha okay so I've just realized I don't actually know what `{ get set }` does, because this works, when I wouldn't expect it to:

```swift
protocol Cookable {
    var temp: Int { get }
}

class Chicken: Cookable {
    var temp = 350
}

let chicken = Chicken()
print(chicken.temp) // prints 350

chicken.temp = 450
print(chicken.temp) // prints 450
```